src/feat_engineering.py

```diff
@@ -1,8 +1,9 @@
 import pandas as pd
 import matplotlib.pyplot as plt
+from tqdm import tqdm
 
 
-def eliminate_long_sentences(data: pd.DataFrame, max_length: int = 400) -> pd.DataFrame:
+def eliminate_long_sentences(data: pd.DataFrame, max_length: int = 80) -> pd.DataFrame:
```

Comment:
why this change in default value? Have you also deleted small sentences?

Reply:
I don't recall that. Is it justified, given hat most of the sentences are under 80, with the mean in both datasets are under 50? Like I don't think we can say it's outliers

Reply:
We discussed it during the lab (I think two weeks ago). You said that most sentences are under 80. We can increase it, but that will also increase the training time. Since you are responsible for training it, I would say it is your call to increase it if you think it is necessary

Reply:
Lmao I must've said it without looking at the boxplot. I think we shouldn't eliminate any short sentences, but also lack of computational resources may be a valid reason why we can eliminate short sentences. At the same time, our model may struggle more without the short sentences, since long sentences have more connectives and manners of speech. Should we ask our TA for advice?

Reply:
maybe we should make an issue and assign it to him? idk if that's ok